docs/using-cli.md

```diff
@@ -14,6 +15,34 @@
 * [Clear cache](#clear-cache)
 * [Check version](#check-version)
 
+## Notes on entity files written in typescript
+This CLI tool is written in javascript and to be run on node. If your entity files are in typescript, you will need to transpile them to javascript before using CLI. You may skip this section if you only use javascript.
+
+You may setup ts-node in your project to ease the operation as follows:
+
+Install ts-node globally:
+```
+npm install -g ts-node
+```
+
+Add typeorm command under scripts section in package.json
+```
+"script" {
+    ...
+    "typeorm": "ts-node -r tsconfig-paths/register ./node_modules/.bin/typeorm"    
```

Comment:
what is `tsconfig-paths/register` here? shouldn't it be installed separately? why do we need users to enforce to register it?

Reply:
Sorry, I am actually using typeorm with nestjs setup and I cannot run properly without registering tsconfig-paths/register (complaining MODULE NOT FOUND)

Reply:
afaik `tsconfig-paths/register` is a way to allow `ts-node` to work with `paths`, I don't think it's really needed to be here

Reply:
btw, shouldn't just `typeorm` work? without `./node_modules/.bin?

Reply:
I just create a fresh project with typeorm only according to the doc. Can confirm that it can run without tsconfig-paths/register.

No idea why it doesn't work on nestjs. Checking...

Reply:
Oh... I found out why now.

In a typical setup, ppl will put entity files under 'src/entities' and register the tsconfig-path somewhere else (like app.ts as suggested in doc). So if the entity references another entity with like OneToOne relation, without tsconfig-paths/register, it will fail with "Cannot find module" error.

It will fail in my fresh typeorm only setup if there is one entity referencing another file.

I think you better keep that in your doc to save your time from answering dumb ppl like me from complaining.

Thanks for this great lib again!

Reply:
> btw, shouldn't just `typeorm` work? without `./node_modules/.bin`?

@havenchyk ppl may not install typeorm globally

Reply:
Okay, I think I found the reason. During the start of npm script, npm adds `./node_modules/.bin` to the path, so if you create a script like `"typeorm": "typeorm"`, it will use `typeorm` from the `.bin` folder, but if `"typeorm": "ts-node typeorm"`, npm will not resolve `./node_moduels/.bin/typeorm`. Even better to use `$(npm bin)` - it's an absolute path to `node_modules/.bin`, so next code will work from any place `"typeorm": "npx ts-node $(npm bin)/typeorm"`

Reply:
What if I use babel instead of ts-node?